.changeset/pink-corners-argue.md

```diff
@@ -0,0 +1,5 @@
+---
+"@liam-hq/schema": patch
+---
+
+🐛 Fix PostgreSQL CHECK constraint parsing with chunk offset correction to prevent malformed constraint details.
```

Comment:
The bug fix changeset should describe what the problem is for the user and what you have solved. The user should not need to know about the chunk.

Reply:
You were right. I rewrote it.🙏
https://github.com/liam-hq/liam/pull/3118/commits/1e8a1a85a9f40d315efb062ea37032931c866e3b

Reply:
I thought it was a very good description! thanks 🚀